src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -89,6 +93,28 @@ export class HuiGlanceCard extends hassLocalizeLitMixin(LitElement)
     }
   }
 
+  public getElementConfig(config: any, hass: HomeAssistant): TemplateResult {
+    if (!config || !hass) {
```

Comment:
Wait a second, this whole method needs to be removed. What is it doing here? There already is `getConfigElement` on Glance Card. All this stuff needs to be in `hui-glance-card-editor`.

We should never conflate concerns. There is an element that is responsible for SHOWING a glance card, there is an element responsible for EDITING a glance card.